### ISSUE TITLE
cmd/govim: investigate tempmodfile flake

### ISSUE THREAD

**Issue by leitzler** [govim/govim#909]
CI action: https://github.com/govim/govim/pull/908/checks?check_run_id=879622445

Log: [12_test (ubuntu-latest, go1.14.4, gvim, v8.2.0931).txt](https://github.com/govim/govim/files/4936740/12_test.ubuntu-latest.go1.14.4.gvim.v8.2.0931.txt)

Artefacts:
* [gopls.log](https://github.com/govim/govim/files/4936729/gopls.log)
* [govim.log](https://github.com/govim/govim/files/4936730/govim.log)
* [vim_channel.log](https://github.com/govim/govim/files/4936731/vim_channel.log)

**Comment by myitcv** [comment]
I'd suggest this can probably be raised as an upstream `gopls` issue for investigation. Are you happy doing that @leitzler?

**Comment by leitzler** [comment]
Yeah, just happened again on another configuration. I'll raise an issue towards `gopls` as soon as I can.

**Comment by leitzler** [comment]
Another: https://github.com/govim/govim/pull/908/checks?check_run_id=881157672
Log: [10_test (ubuntu-latest, go1.14.4, gvim, v8.1.1711).txt](https://github.com/govim/govim/files/4936868/10_test.ubuntu-latest.go1.14.4.gvim.v8.1.1711.txt)

* [gopls.log](https://github.com/govim/govim/files/4936859/gopls.log)
* [govim.log](https://github.com/govim/govim/files/4936860/govim.log)
* [vim_channel.log](https://github.com/govim/govim/files/4936861/vim_channel.log)

**Comment by leitzler** [comment]
https://github.com/golang/go/issues/40269

**Comment by myitcv** [comment]
Awesome, thank you!

**Comment by myitcv** [comment]
Interesting to note that these flakes all appear with `gvim`. Furthermore that when using `vim`, the delete/create pair is not used to "modify" the file, instead a direct modify is used.

But per https://github.com/golang/go/issues/40269#issuecomment-660591285 I think this detail is actually irrelevant.
